_component_design/button-iconography.md

```diff
@@ -0,0 +1,354 @@
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+
+## States
+
+Icon-only buttons can exist in *default*, *hover*, *disabled*, and *focus* states.
+
+Rather than using Cyan 900 to color the tertiary buttons, use `#424242` to make them more subdued and less overwhelming to the user.
+
+See [focus states]({{site.baseurl}}/style/focus-states.html) for guidance in constructing focused state icon buttons.
+
+{% endcolumn %}
+
+{% column right:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+{% figure [caption:"Icon-only button states"] [class:"image bg-light border"] %}
+<embed src="{{site.url}}/assets/images/components/inputs-and-controls/button-iconography/buttoniconography-icononly-states-modtertiary.png" width="270"/>
+{% endfigure %}
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## Examples
+
+This section includes examples of common icon buttons that you can use when constructing a control panel page.
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+
+### Common icon buttons
+
+Choose from a variety of standard icon buttons. See the [Helix Toolkit](https://rackerlabs.github.io/helix-ui/components/icons/) for more information about icons.
+
+{% endcolumn %}
+
+{% column right:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+{% figure [caption:"Common icon buttons"] [class:"image bg-light border"] %}
+<embed src="{{site.url}}/assets/images/components/inputs-and-controls/button-iconography/buttoniconography-icononly-examples-common.png" width="440"/>
+{% endfigure %}
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+
+### Secondary filter
+
+When you add a filter button to a page, ensure it is a secondary button. Do not use a primary button for a filter.
+
+{% endcolumn %}
+
+{% column right:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+{% figure [caption:"Secondary filter buttons"] [class:"image bg-light border"] %}
+<embed src="{{site.url}}/assets/images/components/inputs-and-controls/button-iconography/buttoniconography-icononly-examples-secondary-filter.png" width="633"/>
+{% endfigure %}
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+
+### Cog
+
+The cog is the most prevalent workflow-based icon button used in Rackspace systems. When the user clicks a cog, a drop-down menu of actions appears.  
```

Comment:
Nit: The first sentence here feels like perhaps this is inside info that shouldn't be on the public site. I'd recommend rewriting it if you can to make the move to one-site easier.

Reply:
@palindromebart, including you on this. I'm fine with changing to "The cog is a commonly used workflow-based icon." And drop any mention of its prevalence in our technology.

Reply:
that's fine.  LGTM.